src/bin/sw_single_wrapper

```diff
@@ -8,8 +8,11 @@
 ARGS=""
 for ARG in "$@"
 do
-    QUOTED_ARG=$(printf %q "$ARG")
-    ARGS="$ARGS $QUOTED_ARG"
+    if [ "$ARG" != "%F" ]
```

Comment:
so do you test it and it really pass here "%F" ? that looks strange. How yast can know what is %F? It should be always expanded to list of arguments, not? If XFCE4 does not expand %F, then it is bug in XFCE and not in YaST.

Reply:
> so do you test it and it really pass here "%F" ?

Yes, I did.

If necessary, I could upload _kind of logs_ here.

> If XFCE4 does not expand %F, then it is bug in XFCE and not in YaST.

Taking into account that the previous change in the YaST file was 2 years ago, a bug in XFCE sound reasonable to me.

Let's check it :)

Reply:
what is the most importatnt it is violation of freedesktop specification. So if you can upload logs to bug report, I can redirect it to xfce maintainers.

Reply:
Ok, I'll do